it/controllers/benefits/LivingAccommodationBenefitsControllerISpec.scala

```diff
@@ -0,0 +1,496 @@
+
+      lazy val result: WSResponse = {
+        dropEmploymentDB()
+        userDataStub(userData(fullEmploymentsModel()), nino, taxYearEOY)
+        insertCyaData(employmentUserData(isPrior = false, cyaModel("employerName", hmrc = true,
+          Some(benefits(emptyAccommodationRelocationModel)))), userRequest)
+        authoriseAgentOrIndividual(isAgent = false)
+        urlGet(livingAccommodationPageUrl(taxYearEOY), follow = false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+      }
+
+      s"has a SEE_OTHER($SEE_OTHER) status" in {
+        result.status shouldBe SEE_OTHER
+        //TODO - change to the first page of travel section
+        result.header("location") shouldBe Some(CheckYourBenefitsController.show(taxYearEOY, employmentId).url)
+      }
+
+    }
+
+  }
+
+  ".submit" should {
+
+    userScenarios.foreach { user =>
+
+      import Selectors._
+      import user.commonExpectedResults._
+
+      s"language is ${welshTest(user.isWelsh)} and request is from an ${agentTest(user.isAgent)}" should {
+
+        "return an error when a form is submitted with no entry" which {
+
+          implicit lazy val result: WSResponse = {
+            authoriseAgentOrIndividual(user.isAgent)
+            dropEmploymentDB()
+            userDataStub(userData(fullEmploymentsModel()), nino, taxYearEOY)
+            insertCyaData(employmentUserData(isPrior = true, cyaModel("name", hmrc = true, Some(benefits(
+              fullAccommodationRelocationModel)))), userRequest)
+            urlPost(livingAccommodationPageUrl(taxYearEOY), body = "", welsh = user.isWelsh,
+              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
+          }
+
+          s"has an BAD REQUEST($BAD_REQUEST) status" in {
+            result.status shouldBe BAD_REQUEST
+          }
+
+          implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+          titleCheck(user.specificExpectedResults.get.expectedErrorTitle)
+          h1Check(user.specificExpectedResults.get.expectedHeading)
+          captionCheck(expectedCaption(taxYearEOY), captionSelector)
+          errorSummaryCheck(user.specificExpectedResults.get.expectedErrorMessage, yesSelector)
+          formPostLinkCheck(continueButtonLink, formSelector)
+
+        }
+      }
+    }
+
+    "update accommodationQuestion to Some(true) when user chooses yes and it's a prior submission" which {
+
+      lazy val form: Map[String, String] = Map(YesNoForm.yesNo -> YesNoForm.yes)
+
+      implicit lazy val result: WSResponse = {
+        authoriseAgentOrIndividual(isAgent = false)
+        dropEmploymentDB()
+        userDataStub(userData(fullEmploymentsModel()), nino, taxYearEOY)
+        insertCyaData(employmentUserData(isPrior = true, cyaModel("employerName", hmrc = true,
+          Some(benefits(fullAccommodationRelocationModel)))), userRequest)
```

Comment:
For this test, the accomodationQuestion was already set to true in the cya data. It might be worth making a new model where its false, so that when you do your submission its going from false to true.